pycbc/waveform/bank.py

```diff
@@ -68,6 +68,11 @@ class BaseFilterBank(object):
     about elements of an xml template bank.
     """
     def __init__(self, filename, approximant=None, **kwds):
+
```

Comment:
Indeed this modification looks pretty convoluted and backwards. However I found no other way to make this work with BOINC's handling of files communicated with the server ("input" and "output" files), the limited abilities of the BOINC "wrapper" program to manipulate the command-line of the program it calls, and the fact that a Cygwin Pyinstaller bundle can't (relatively) reference files outside the "root" directory of the bundle.